src/schemas/pa-yaml/v3.0/pa.schema.yaml

```diff
@@ -230,6 +230,9 @@ definitions:
         Variant:
           type: string
           minLength: 1
+        Layout:
```

Comment:
this is correct!
To finalize, you need to manually run: `src\schemas\publish.cmd` and it'll compile and update the published schema. check that change in too.